example/server

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env perl6
+
+use Stomp::Server;
+
+sub MAIN(:$port = 61613, :$host = 'localhost') {
+    my $server = Stomp::Server.new(:$port, :$host);
+
+    my $all-messages = Supplier.new;
+    react {
+        whenever $server.listen -> $v {
+            whenever $v.sent-messages -> $m {
```

Comment:
Hm, not sure whether `sent-messages` is quite right, in so far as we're writing a server, from whose perspective the messages are received. On the other hand, the command the client issues is `SEND`. Tricky. :-)